src/Avatar/Avatar.js

```diff
@@ -4,24 +4,52 @@ import PropTypes from 'prop-types';
 import { Avatar as CoreAvatar } from 'wix-ui-core/dist/src/components/avatar';
 import { avatar as avatarStyle } from 'wix-ui-core/dist/src/themes/backoffice';
 
-import style from './Avatar.st.css';
+import styles from './Avatar.st.css';
 import { capitalize } from '../utils/cssClassUtils';
 
+const sizeNumber = size => Number(size.substring(4));
+
 /**
  * Avatar is a type of element that visually represents a user, either as an image, name initials or placeholder icon.
  */
 const Avatar = props => {
-  const { size, color, dataHook, className, ...rest } = props;
-
+  const {
+    size,
+    presence,
+    indication,
+    color,
+    onClick,
+    dataHook,
+    className,
+    ...rest
+  } = props;
+  const renderIndication = indication && sizeNumber(size) > 36;
   return (
-    <CoreAvatar
-      {...rest}
-      data-hook={dataHook}
-      className={classNames(
-        className,
-        avatarStyle(size, color && `color${capitalize(color)}`),
-      )}
-    />
+    <div className={className}>
+      <div {...styles('avatarContainer', { size, indication, presence })}>
+        <div className={styles.avatarPresenseContainer}>
+          <div {...styles('coreAvatar', { size, presence })}>
+            <CoreAvatar
+              {...rest}
+              data-hook={dataHook}
+              className={classNames(
+                avatarStyle(size, color && `color${capitalize(color)}`),
+              )}
+            />
+          </div>
+          {presence && <div {...styles('presence', { size, presence })}></div>}
+        </div>
+        {renderIndication && (
+          <div
+            data-hook="indication"
```

Comment:
Please change to `avatar-indication` so the dataHook will be unique:
```suggestion
            data-hook="avatar-indication"
```